src/ResponseLocation/XmlLocation.php

```diff
@@ -80,7 +80,7 @@ public function visit(
     ) {
         $sentAs = $param->getWireName();
         $ns = null;
-        if (strstr($sentAs, ':')) {
+        if (strstr($sentAs ?: '', ':')) {
```

Comment:
`?:` is a bit of a hack. Can we do an explicit null check please?

```
if (null !== $sentAs && strstr($sentAs, ':')) {
```

Reply:
done.